### ISSUE TITLE
New buttons

### ISSUE THREAD

**Issue by Ijin08** [grafana/grafana#15442]
### Motivation

Our current buttons are suffering by some issues such as inconsistency(which has been mostly solved in master now) inaccessibility, not the optimal naming in the code.     
To improve on this we should define the different actions the buttons are used for and name the after that. We should make sure the the buttons are accessible and appealing. I do really think the buttons should be accessible and live up to WCAG AA.

### Overview: 

The buttons can be placed in three different categories: 

**Toolbar buttons**

These buttons are used on the Dashboard, in Panel editor and in Explore.
They are neutral and do not stand out. They shouldn’t color wise compete with the colors in the panels.

**Settings buttons**

These buttons are found in dashboard settings, the configuration pages and to some extent in panel edit. These buttons have a primary action that stands out. Sometimes there is a secondary action that will be neutral colored so not to compete with primary. 

My suggestion for the setting buttons:

I suggest making the add/save button blue simply because it makes it easier to make an accessible and appealing button. With green you either get a nice vibrant button with low contrast or a dark and boring button with good contrast. Both blue and green are positive colors when it comes to buttons. Blue is the most common color for CTAs and the most popular color. 

For the neutral button used for secondary actions I suggest adding a bit more contrast against the background. I also suggest reversing the gradient so all buttons are consistent. 

The delete button has already been implemented.

For cancel I suggest simply to just use a link. 

<img width="796" alt="screenshot 2019-02-14 at 15 54 24" src="https://user-images.githubusercontent.com/23470681/52794817-da647d80-3070-11e9-94d0-0af570f1b0d4.png">

**Run buttons**

Currently only used in Explore. This should be a primary button that differs from the settings buttons. It should also not distract from the content.  

My suggestion for run button is to use a less saturated color than the ones in settings.
Colors I am leaning towards are purple and teal since they are easy to work with when it comes to contrast. 

To do:

- [x] Decide on color for add/save in settings
- [x] Decide on color for run in Explore
- [x] Give buttons intuitive names

**Comment by Ijin08** [comment]
@davkal @torkelo 
Would like your thoughts on this.

**Comment by torkelo** [comment]
I am fine with this. Think we stay with green as CTA for now. Like to change to have Cancel be a link, and to try a less saturated color for secondary CTA.

**Comment by dprokop** [comment]
@torkelo @Ijin08 @davkal FYI I have started migration of buttons to @grafana/ui here https://github.com/grafana/grafana/pull/16228

**Comment by davkal** [comment]
Can this issue be closed @Ijin08 ?

**Comment by Ijin08** [comment]
Will close this issue. We will probably revisit button design in the future.
